metrics/core/metrics.go

```diff
@@ -244,7 +250,7 @@ var MetricCpuUsage = Metric{
 
 var MetricEphemeralStorageUsage = Metric{
 	MetricDescriptor: MetricDescriptor{
-		Name:        "ephemeralstorage/usage",
+		Name:        "ephemeral_storage/usage",
```

Comment:
New metrics should be added to https://github.com/kubernetes/heapster/blob/master/docs/storage-schema.md#metrics

Reply:
added. Thanks!